src/protocol.js

```diff
@@ -70,4 +70,13 @@ module.exports = class Protocol {
         return Array.from(unresolvedRefs);
     }
 
+    getSuccessCodeOutputKey(domainId) {
+        const domain = this.getDomain(domainId);
+        return domain ? domain.getSuccessCodeOutputKey() : null;
+    }
+
+    getSuccessCodeValue(domainId) {
+        const domain = this.getDomain(domainId);
+        return domain ? domain.getSuccessCodeValue() : null;
```

Comment:
This passes `undefined` as `outputData` param (we should switch to TypeScript :D )

Reply:
Thanks for spotting, fixed! and fixed wrong tests too:D